README.md

```diff
@@ -1,4 +1,14 @@
 # NF1_SchwannCell_data_analysis
+## Repository Structure
```

Comment:
Recommend including a short description of this repo in between these two sections. I think you include it after the `Repository Structure` title, so it might be better moved above it.

Reply:
I restructured this a little. Lmk what you think!